src/Datadog.Trace.ClrProfiler.Native/logging.cpp

```diff
@@ -22,6 +39,13 @@ std::string Logger::GetLogPath() {
       std::filesystem::create_directories(parent_path);
     }
   }
+#else
+  // on linux we use the basic C approach
+  const auto log_path = getPathName(path);
+  Stat st;
+  if (stat(log_path.c_str(), &st) != 0) {
```

Comment:
Shouldn't you skip this when log_path is "" ?

Reply:
I wonder if is "" possible in a linux path?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1028/commits/3cde2d05b3fc36620330f56732b9b99154586d22